Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -247,6 +247,63 @@ function Invoke-MonitorShallow($authorizationUrl) {
     Invoke-RestMethod -Method Get -Uri $url
 }
 
+function Get-EdwAdminUsersAndGroups($connectionString) {	
+    $connection = New-Object System.Data.SqlClient.SqlConnection($connectionString)	
+    $sql = "SELECT i.IdentityID, i.IdentityNM, r.RoleNM	
+            FROM [CatalystAdmin].[RoleBASE] r	
+            INNER JOIN [CatalystAdmin].[IdentityRoleBASE] ir	
+                ON r.RoleID = ir.RoleID	
+            INNER JOIN [CatalystAdmin].[IdentityBASE] i	
+                ON ir.IdentityID = i.IdentityID	
+            WHERE RoleNM = 'EDW Admin'";	
+    $command = New-Object System.Data.SqlClient.SqlCommand($sql, $connection)	
+    
+    $usersAndGroups = @();	
+    try {	
+        $connection.Open()    	
+        $reader = $command.ExecuteReader()	
+        while ($reader.Read()) {	
+            $usersAndGroups += $reader['IdentityNM']	
+        }	
+        $connection.Close()        	
+    
+    }	
+    catch [System.Data.SqlClient.SqlException] {	
+        Write-Error "An error ocurred while executing the command. Please ensure the connection string is correct and the metadata database has been setup. Connection String: $($connectionString). Error $($_.Exception.Message)"  -ErrorAction Stop	
+    }    	
+    
+    return $usersAndGroups;	
+}	
+    	
+function Add-ListOfUsersToDosAdminRole($edwAdminUsers, $domain, $connString, $authorizationServiceUrl, $accessToken) {	   
```

Comment:
Since we aren't validating users against the domain, should we drop the $domain parameter?

Reply:
yep, we can remove that.